ssh-pkcs11.h

```diff
@@ -22,10 +22,14 @@
 #define	SSH_PKCS11_ERR_PIN_REQUIRED		4
 #define	SSH_PKCS11_ERR_PIN_LOCKED		5
 
-int	pkcs11_init(int);
-void	pkcs11_terminate(void);
-int	pkcs11_add_provider(char *, char *, struct sshkey ***, char ***);
-int	pkcs11_del_provider(char *);
+struct sshkey * lookup_key(const struct sshkey *);
```

Comment:
Please move the windows specific code to #ifdef WINDOWS block. Please add comments why there is a deviation.